docs/onboarding/Configuration-Properties.md

```diff
@@ -144,7 +144,6 @@ The Onboarding Server uses the following public configuration properties:
 | `enrollment-server-onboarding.provider.innovatrics.serviceUserAgent`                                    | `Wultra/OnboardingServer`                                                                                 | User agent to use when making HTTP calls to Innovatrics REST service.                                                                                                                                                                                              |
 | `enrollment-server-onboarding.provider.innovatrics.presenceCheckConfiguration.score`                    | 0.875                                                                                                     | Presence check minimal score threshold.                                                                                                                                                                                                                            |
 | `enrollment-server-onboarding.provider.innovatrics.documentVerificationConfiguration.documentCountries` | `CZE`                                                                                                     | List of expected countries of issue of identification documents as three-letter country codes, i.e. ISO 3166-1 alpha-3. If empty, all countries of issue known to Innovatrics are considered during classification, which may have negative impact on performance. |
-| `enrollment-server-onboarding.provider.innovatrics.document-verification-configuration.crucialFields`   | `documentNumber`, `dateOfIssue`, `dateOfExpiry`, `surname`, `dateOfBirth`, `personalNumber`, `givenNames` | Set of fields in camelCase that are cross-validated between the machine-readable zone and visual zone. Only those specified fields that are actually extracted from a document are considered. If empty, no inconsistencies will be checked.                       |
```

Comment:
I am wondering where and how to document  Innovatrics instance configuration. I do not mean to copy&paste their documentation, but highlight keywords and record our expectations.

Reply:
Created dedicated issue https://github.com/wultra/enrollment-server/issues/964